doc/INSTALL.txt

```diff
@@ -618,11 +618,11 @@
   located in the doc directory. It's named INSTALL-MacOS.txt.
 
 -------------------------------------------------------------------------------
-  9. Compiling on Win32
+  9. Compiling on Windows
 -------------------------------------------------------------------------------
 
-  There is a detailed compilation and installation HOWTO for Win32 systems
-  located in the doc directory. It's named INSTALL-Win32.txt.
+  There is a detailed compilation and installation HOWTO for Win32 and Win64 systems
+  located in the doc directory. They're named INSTALL-Win32.txt and INSTALL-Win64.txt.
```

Comment:
+1 for not duplicating information,

Make a readme named INSTALL-Windows.txt wack everything in there and document any differences.